### PR TITLE
Share dialog

### LINKED ISSUE #7977: Implement share dialog
screen shot 2016-06-21 at 09 54 48

cc @gfiorav

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes #7977.

share

It includes the update endpoint on Mapcaps, but not sure how to proceed once it fails or succeeds. Also, in the case of Safari, because the copy plugin is not fully working, I've changed the label for `SELECT` instead, leaving the user the responsibility of `cmd + C`. I left all the cards, even some of them will be out at the beginning.

safari

@piensaenpixel all the svgs icon need to be included, and please, take a look to the styles.

cc @xavijam @gfiorav

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/modals/share/share-view.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    });
+    this.$('.js-info').append(view.render().el);
+    this.addView(view);
+  },
+
+  _onContinue: function () {
+    this._modalModel.destroy();
+  },
+
+  _onUpdate: function () {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/share/share-view.js]: mapcap too

File: lib/assets/javascripts/cartodb3/data/vis-definition-model.js [hunk 2]
```diff
@@ -18,6 +18,16 @@ module.exports = Backbone.Model.extend({
     var baseUrl = this._configModel.get('base_url');
     var version = this._configModel.urlVersion('visualization');
     return baseUrl + '/api/' + version + '/viz';
+  },
+
+  mapcapsURL: function () {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/data/vis-definition-model.js]: ^^^

File: lib/assets/javascripts/cartodb3/components/modals/share/info/info.tpl [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div class="CDB-Text CDB-Size-medium">
+  <%= updatedOn %>
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/share/info/info.tpl]: <%= %> or <%- %>?

**Comment by xavijam** [comment]
👍  🚢 🇮🇹  when you consider it is done :)

**Comment by nobuti** [comment]
@xavijam ir should be nice if you double check this part https://github.com/CartoDB/cartodb/blob/5f1a1bbd6919e78f78a72663ab77496c4af4abd5/lib/assets/javascripts/cartodb3/data/vis-definition-model.js#L27-L41, not sure if I'm missing something permission related.

**Comment by xavijam** [comment]
I feel it is ok, there are permission problems in the table context, in the visualization one we are "kind" of safe.

**Pull request merged**